pegasus/migrations/106_add_popularityrank_to_tutorials.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :popularityrank, Integer, null: false
```

Comment:
We just pull down the entire table when opening the /learn page, so I think we're okay without an index.